### ISSUE TITLE
Failure in Network Training

### ISSUE THREAD

**Issue by Arafat4341** [espnet/espnet#1667]
I am trying to run the docker image of espnet on t2.micro ec2 instance of AWS.
I ran this command: `./run.sh --docker_gpu -1 --docker_egs an4/asr1   --docker_user false  --ngpu 0`

Job fails at Stage 4: Network Training. I get this:

`stage 4: Network Training
bash: line 1:  6508 Killed
( asr_train.py --config conf/train_mtlalpha1.0.yaml --ngpu 0 --backend pytorch --outdirexp/train_nodev_pytorch_train_mtlalpha1.0/results --tensorboard-dir tensorboard/train_nodev_pytorch_train_mtlalpha1.0 --debugmode 1 --dictdata/lang_1char/train_nodev_units.txt --debugdir                    exp/train_nodev_pytorch_train_mtlalpha1.0 --minibatches 0 --verbose 1 --resume -                   -train-json dump/train_nodev/deltafalse/data.json --valid-json dump/train_dev/de                   ltafalse/data.json ) 2>> exp/train_nodev_pytorch_train_mtlalpha1.0/train.log >>                    exp/train_nodev_pytorch_train_mtlalpha1.0/train.log

run.pl: job failed, log is in exp/train_nodev_pytorch_train_mtlalpha1.0/train.log
run.sh done.`

Why is this occurring?
Is my machine good enough to run the docker image?

**Comment by sw005320** [comment]
Please check `exp/train_nodev_pytorch_train_mtlalpha1.0/train.log` to see what happened.

**Comment by Arafat4341** [comment]
I ran `run.sh`.  It exceeded the `stage 4: Network Training` and went to `stage 5: Decoding`.
Finally It completed and gave a `run.sh done.` message.
Then I went on to test a pretrained model on ljspeech dataset. I tried this command: `../../../utils/synth_wav.sh example.txt` on an example text I created. I get `The standard file /home/arafat/espnet/egs/ljspeech/tts1/../../../tools/kaldi/tools/config/common_path.sh
is not present -> Exit!`

I didn't install kaldi. I ran the docker image of espnet and I thought I shall get kaldi installed if I run docker image of `espnet`. Am I right about this?
I am closing this issue by the way.
